Vizzuality/heco-invest
# API endpoint for locations

I am adding two new API endpoints for location --> one for `index` and second for `show/detail`.

Only other thing which is probably worthy to mention is that `index` locations API endpoint allows to filter locations based on:
 - location_type
 - parent_id

I have also added LocationType values to Enum ;).

## Testing instructions

Please check swagger documentation, there are two new api endpoints for locations:
![Screenshot 2022-03-21 at 11 42 21](https://user-images.githubusercontent.com/20660167/159245592-49ca7a2e-f207-46b3-bf39-1fb75fc0103b.png)

```
{
  "data": {
    "id": "4debca7f-9251-403f-b893-ff0e78bd6af4",
    "type": "location",
    "attributes": {
      "name": "Hane, Lehner and Goyette",
      "location_type": "municipality"
    },
    "relationships": {
      "parent": {
        "data": {
          "id": "5331a4ba-ef75-4b7f-8cdd-8ec39e692bbd",
          "type": "location"
        }
      },
      "regions": {
        "data": [
          {
            "id": "b5debe16-a4ab-4811-ac48-3b480af1fc56",
            "type": "location"
          }
        ]
      }
    }
  }
}
```

## Tracking

https://vizzuality.atlassian.net/browse/LET-129
